Assets/_Laz/Scripts/Managers/WorldManager.cs

```diff
@@ -99,17 +101,32 @@ private IPlanetoidBehaviour[] GenerateObjectOfInterest(GameObject[] interests)
             return interests.Select(x => x.GetComponent<IPlanetoidBehaviour>()).ToArray();
         }
 
-        private void HandleOnStateChanged(State state)
+        private void SetDeathState()
         {
-            if (state == State.PreGame)
-            {
-                CleanUp(); 
-                StateManagerInstance.SetState(State.Play);
-            }
+            StateManagerInstance.SetState(State.Death);
+        }
 
-            if (state == State.Play)
+        private void HandleOnStateChanged(State state)
+        {
+            switch (state)
             {
-                Reset();
+                case State.PreGame:
+                {
+                    CleanUp();
+                    StateManagerInstance.SetState(State.Play);
+                    break;
+                }
+                case State.Play:
+                {
+                    Reset();
+                    break;
+                }
+                case State.Death:
+                {
+                    Debug.LogWarning("Laz is dead, long live Laz!");
+                    StateManagerInstance.SetState(State.PreGame);
```

Comment:
So because you changed it so that you did a reset, can you update the ticket, so that within the Acceptance Criteria, to tell Kevin to test that it resets?